### DISCUSSION TITLE
怎样设置才能不翻译繁体字幕

### DISCUSSION THREAD

**Discussion by OF65FUpwyvvy** [immersive-translate/immersive-translate#1040] [Help]
### 概述 | Summary

我不需要翻译繁体字幕，我的设置如下，但没什么用，
![chrome_TkC7CRiOVZ](https://github.com/immersive-translate/immersive-translate/assets/70679435/17a3ecae-7ff8-4ca9-8d9e-09dd33709c1a)
![chrome_E3DPJnbk47](https://github.com/immersive-translate/immersive-translate/assets/70679435/f69f19eb-2942-4c1d-ab38-06da9362772a)
每次打开有繁体字幕的视频还是会自动开启翻译，如下图
![chrome_mWFymnaXBP](https://github.com/immersive-translate/immersive-translate/assets/70679435/f89c0fa2-3d29-417e-9c75-1d9a548279b3)
要怎样设置才能不翻译繁体字幕。

**Comment by theowenyoung** [comment]
你好，最新版本0.10.7 已经优化了该问题，当用户的目标语言为中文的时候，那么默认不翻译 繁体或简体中文。
